api/swagger/swagger-dev.json

```diff
@@ -20548,6 +20548,129 @@
         }
       }
     },
+    "/v1/management/backup/Locations/Change": {
+      "post": {
+        "tags": [
+          "Locations"
+        ],
+        "summary": "ChangeLocation changes backup location.",
+        "operationId": "ChangeLocation",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "location_id": {
+                  "description": "Machine-readable ID.",
+                  "type": "string",
+                  "x-order": 0
+                },
+                "name": {
+                  "type": "string",
+                  "title": "Location name",
+                  "x-order": 1
+                },
+                "description": {
+                  "type": "string",
+                  "x-order": 2
+                },
+                "pmm_client_config": {
+                  "description": "PMMClientLocationConfig represents file system config inside pmm-client.",
+                  "type": "object",
+                  "properties": {
+                    "path": {
+                      "type": "string",
+                      "x-order": 0
+                    }
+                  },
+                  "x-order": 3
+                },
+                "pmm_server_config": {
+                  "description": "PMMServerLocationConfig represents file system config inside pmm-server.",
+                  "type": "object",
+                  "properties": {
+                    "path": {
+                      "type": "string",
+                      "x-order": 0
+                    }
+                  },
+                  "x-order": 4
+                },
+                "s3_config": {
+                  "description": "S3LocationConfig represents S3 bucket configuration.",
+                  "type": "object",
+                  "properties": {
+                    "endpoint": {
+                      "type": "string",
+                      "x-order": 0
+                    },
+                    "access_key": {
```

Comment:
I guess we need the bucket name as well, as per 4.5 of BackupManagement Architecture.